tests/standalone/PyparsingUsing.py

```diff
@@ -0,0 +1,15 @@
+#program to parse any greeting of the form <salutation>,<address>!
+
+from pyparsing import Word, alphas
+
+# nuitka-skip-unless-imports: Word, alphas
+
+greet = Word(alphas) + "," + Word(alphas) + "!"
+greeting = greet.parseString("Hello , World !")
+print(greeting)
+
+def good_parse():
```

Comment:
You can structure your code in a slightly different way. You can create a class called `PyparsingGreeting`, create the `greet` object in the `__init__` method, rename the `good_parse` method to `test_greeting` and write the logic there. In the `if __name__ == '__main__'` block, you can create an instance of `PyparsingGreeting` and call the `test_greeting` method.